vendor/engines/split_accounts/app/services/split_accounts/order_detail_splitter.rb

```diff
@@ -78,9 +80,9 @@ def apply_order_detail_remainders
     end
 
     def apply_reservation_remainders
-      return unless order_detail.reservation
-      applier = RemainderApplier.new(order_detail.reservation, split_order_details.map(&:reservation), splits)
-      applier.apply_remainders(reservation_attribute_splitter)
+      return unless order_detail.time_data.present?
```

Comment:
not necessarily on this specific line.. but the `time_data` naming is kind of confusing to me since it's sort-of "data about time" and also sort-of "either an occupancy or reservation attached to this order detail", should we rename / maybe have a separate method? I'm not sure what I'd do